system/ee/EllisLab/Addons/member/mod.member_settings.php

```diff
@@ -1658,7 +1658,7 @@ function do_member_mini_search()
 			{
 				if ($val != '')
 				{
-					$search_query[] = $key." LIKE '%".ee()->db->escape_like_str($val)."%'";
+					$search_query[] = ee()->db->escape_like_str($key)." LIKE '%".ee()->db->escape_like_str($val)."%'";
```

Comment:
Shouldn't this use `escape_str` instead?

Reply:
Yeah, good catch - I didn't realize there were two functions and let autocomplete fill in that function when I was typing it. I'm updating it and will resolve this conversation once it is pushed up.